packages/app/src/__deprecated__/renderer/wrappers/root-wrapper.tsx

```diff
@@ -202,6 +205,14 @@ const RootWrapper: FunctionComponent<Props> = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
+  useEffect(() => {
+    if (deviceType) {
+      if (deviceType === DeviceType.MuditaHarmony) {
```

Comment:
I think it could be single if not two, but it depends on convention, it's fine :)